bots/webcomic.bot.js

```diff
@@ -35,22 +35,23 @@ const getComics = (arr, count=3)=>{
 Slack.onMessage(async (msg)=>{
 	try{
 		if(msg.text.startsWith('comic:')){
-			const link = msg.text.split('comic:')[1].trim().replace('<', '').replace('>', '');
-
-			await Gist.append(GistId, {
-				webcomics : [{
-					date: (new Date()).toISOString(),
-					link,
-					user : msg.user,
-					shared : false
-				}]
-			})
-			Slack.send(msg.channel, 'Webcomic saved!');
+			const match = msg.text.match(/<(.+)\|.*>/);
```

Comment:
According to the mrkdwn docs you linked, links apparently _can_ still exist without the pipe in some scenarios. I would recommend a more permissive regex to capture both cases, maybe something like `<([^>\|]+)(>|\|)`. I have not tested that regex. 😬